src/lib/ai/tools/skip.ts

```
@@ -0,0 +1,23 @@
+import { createLogger } from '@/lib/logger';
+import { tool } from 'ai';
+import type { Message } from 'discord.js';
+import { z } from 'zod';
```

ESLint

[error] Unable to resolve path to module 'zod'.

(import-x/no-unresolved)